5g-control-plane/.gitignore

```diff
@@ -1,3 +1,4 @@
+# SPDX-FileCopyrightText: 2024-present Aether Project (Part of Linux Foundation) <dev@lists.aetherproject.org>
 # SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
What about removing the "old" email address from all copyright headers to avoid confusions?
```suggestion
# SPDX-FileCopyrightText: 2020-present Open Networking Foundation
```

Reply:
Fixed